x-pack/test/functional/apps/security/users.ts

```diff
@@ -115,5 +118,104 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
       expect(roles.monitoring_user.reserved).to.be(true);
       expect(roles.monitoring_user.deprecated).to.be(false);
     });
+
+    describe('edit', function () {
+      before(async () => {
+        await PageObjects.security.clickElasticsearchUsers();
+      });
+
+      describe('update user profile', () => {
+        it('when submitting form and redirects back', async () => {
+          const user = {
+            username: 'OptionalUser',
+            full_name: 'Optional User',
+            email: 'optionalUser@elastic.co',
+            roles: [],
+          };
+          await PageObjects.security.updateUser(user);
+          const users = keyBy(await PageObjects.security.getElasticsearchUsers(), 'username');
+
+          expect(users.OptionalUser.roles).to.eql(['']);
+          expect(users.OptionalUser.fullname).to.eql(user.full_name);
+          expect(users.OptionalUser.email).to.eql(user.email);
+          expect(users.OptionalUser.reserved).to.be(false);
+          expect(await browser.getCurrentUrl()).to.contain('app/management/security/users');
+        });
+      });
+
+      describe('change password', () => {
```

Comment:
Note Reviewer : I am worrying about these two tests because I used the toaster mechanism to see if it has been successful or not. We will see if the flaky suite see a problem or not 🤞 🤞

Reply:
the kibana flaky suite did not fail at all here, So I feel really confortable with it ;)